From ecfa5156ba1e03cbe74be2141f9d9587e362c7a8 Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Thu, 22 Oct 2026 17:23:01 +0800
Subject: [PATCH 3/4] FROMLIST: fix the help message of -j

When using -j we need to specify a directory (usually named "java"),
but the example showed the file ("cacerts") in the directory.  Thus if
you follow the example you'd get a "java/cacerts/cacerts" file.

Link: https://github.com/lfs-book/make-ca/pull/42
---
 make-ca | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/make-ca b/make-ca
index f020686..b26b23b 100644
--- a/make-ca
+++ b/make-ca
@@ -296,7 +296,7 @@ function showhelp(){
   echo "                         The output directory for the OpenSSL trusted"
   echo "                         CA certificates"
   echo ""
-  echo "        -j, --javacerts [\$PKIDIR/java/cacerts]"
+  echo "        -j, --javacerts [\$PKIDIR/java]"
   echo "                         The output directory for the Java cacerts"
   echo "                         file(s)"
   echo ""
-- 
2.56.0

